packages/types/lib/responses.ts

```diff
@@ -18,7 +19,14 @@ import {
   Video
 } from "@lib/types";
 
-// Response that should always return an error, used by westegg
+//
+// Generic Responses
+// --------------------------------------------------
+export type CursorResponse = {
+  cursor?: ObjectID;
+  limit?: number;
+};
```

Comment:
For paginated responses, I would prefer if we didn't mix cursor related fields from data fields, so in that case it would be preferable to make this generic (and moved to below the definition of the `Response` type):

```
export type CursorResponse<T = {}> = { content: T; cursor: ObjectID; limit: number; } | ErrorResponse;
```

Reply:
In addition, the new type isn't being used below for the response types of listing routes.